In [2]:
import pandas as pd

In [24]:
subset_pat = pd.read_json('../../Annotation/Patrick/batch2_patrick_result.json')
#subset_thom = pd.read_json('../../Annotation/Thom/test_batch_thom.json')
subset_chiel = pd.read_json('../../Annotation/Chiel/batch1_chiel_result.json')
subset_bar = pd.read_json('batch1_result.json')

In [25]:
subset_pat[subset_pat.vocop_match != 0]

uuid  rubriek      notaris  \
2049  87d8b2f4-b586-8b52-5428-5b7fd24a6e8b      358  JAN VERLEIJ   
2071  e0d5cb34-cd51-5dae-cb7d-83aaf62f32a3      358  JAN VERLEIJ   
2282  b30c8a95-e5a2-f818-da9f-d19ec93d59e6      358  JAN VERLEIJ   

      inventarisNr  akteNr    akteType    datering        taal  \
2049         11908   12813   Testament  1749-09-11  nederlands   
2071         11908   14377  Machtiging  1749-09-08  nederlands   
2282         11910   17760  Machtiging  1750-04-17  nederlands   

                                          beschrijving  \
2049        \nschip Bloemendaal, VOC kamer Amsterdam\n   
2071                              \nschip Sloterdijk\n   
2282  \nSchip de Ouderkerk, schip de Standvastigheid\n   

                                                  namen  \
2049  [{'voornaam': 'Leendert', 'tussenvoegsel': Non...   
2071  [{'voornaam': 'Jacobus', 'tussenvoegsel': None...   
2282  [{'voornaam': 'Johannes', 'tussenvoegsel': 'va...   

                                                   urls  \
2049  ['KLAB05596000423.JPG', 'KLAB05596000424.JPG',...   
2071  ['KLAB05596000398.JPG', 'KLAB05596000399.JPG',...   
2282  ['KLAB05289000441.JPG', 'KLAB05289000442.JPG',...   

                                                   text                  name  \
2049  :15Calkoen Dit Testament vervat No: 437 geen f...  Leendert Zuijderland   
2071  No: 429 Procumatie gecasseerd den 8e: fept: 17...         Hermanus Soet   
2282  No: 137 Procuratie gepasseert den 17e: april 1...              Jan Helt   

                                           data_matches  \
2049  [{'index': 348698, 'name_original': 'Leendert ...   
2071  [{'index': 174610, 'name_original': 'Harmanus ...   
2282  [{'index': 415, 'name_original': 'Jan Held', '...   

                                             data_entry  \
2049  {'348698': {'ships': ['Bloemendaal'], 'rank': ...   
2071  {'174610': {'ships': [], 'rank': [], 'location...   
2282  {'415': {'ships': [], 'rank': [], 'location': ...   

                          vocop_match  
2049  [Leendert Suijderlandt, 348698]  
2071          [Hermanus Soet, 374000]  
2282               [Jan Helt, 417031]

In [23]:
subset_bar[subset_bar.vocop_match != 0]

uuid  rubriek      notaris  \
5668  16106777-6e18-ce1d-f4a0-650fb3247e3a      358  JAN VERLEIJ   
5687  73b36840-adfa-0db8-9d3f-e870cbe275d5      358  JAN VERLEIJ   
5694  82e96f32-75e0-e937-5c7e-5a3e5c0a6792      358  JAN VERLEIJ   
5846  934d3c19-5d20-33d8-5826-9f46658b64b2      358  JAN VERLEIJ   
5857  a517338b-8389-d6f7-18c8-29b1f8f944fc      358  JAN VERLEIJ   
5875  197025df-61b1-730c-efc4-a88fd46ec6fc      358  JAN VERLEIJ   
5964  1bf4e148-14a4-dd4d-ef24-503a157766dc      358  JAN VERLEIJ   
6001  c2a57380-363a-18cd-4a39-621fbc76345f      358  JAN VERLEIJ   
6044  4a742b73-d2fb-cf56-9d0e-1195882de934      358  JAN VERLEIJ   
6072  c32d6954-f53a-68ac-8eb1-9ae1ffff1080      358  JAN VERLEIJ   
6106  e0def5f5-9bca-6182-7b1c-af1fd770663c      358  JAN VERLEIJ   
6190  1766dc6b-87fe-f838-d93e-4251e0011991      358  JAN VERLEIJ   

      inventarisNr  akteNr    akteType    datering        taal  \
5668         11955   21595  Machtiging  1764-10-23  nederlands   
5687         11955   26890  Machtiging  1764-09-29  nederlands   
5694         11955   27743  Machtiging  1764-11-05  nederlands   
5846         11957   28686  Machtiging  1765-07-01  nederlands   
5857         11957   29685  Machtiging  1765-07-06  nederlands   
5875         11958   21795  Machtiging  1765-09-21  nederlands   
5964         11960   21944  Machtiging  1766-08-25  nederlands   
6001         11960   31329  Machtiging  1766-08-12  nederlands   
6044         11961   24564  Machtiging  1766-09-30  nederlands   
6072         11961   31357   Kwitantie  1766-11-18  nederlands   
6106         11961   33040   Testament  1766-09-24  nederlands   
6190         11963   21670  Machtiging  1767-07-20  nederlands   

                                           beschrijving  \
5668  \nVOC schip Pijlswaert, schip De Drie Papagaij...   
5687  \nVOC schip Westerveld, innen van uitstaande g...   
5694  \nVOC schip Cattendijk, schip De vrouw Elisabe...   
5846  \nSchip Overschie; schip de Gouverneur Generaal\n   
5857  \nJoseph vertrok in 1761 aan boord van het VOC...   
5875  \nSchip de Getrouwigheijd voor de Kamer Hoorn;...   
5964  \nVOC schip Damzigt, schip Walcheren, innen va...   
6001  \nKomber vertrok in 1765 aan boord van het VOC...   
6044    \nVOC schip Pallas, machtiging aan zijn vrouw\n   
6072             \nSchip de Vrouwe Elisabeth Adriana.\n   
6106                                               None   
6190  \nMulder was onlangs met het VOC-schip de Jonk...   

                                                  namen  \
5668  [{'voornaam': 'Jan', 'tussenvoegsel': None, 'a...   
5687  [{'voornaam': 'Evert', 'tussenvoegsel': None, ...   
5694  [{'voornaam': 'Cornelis', 'tussenvoegsel': 'de...   
5846  [{'voornaam': 'Ernst August', 'tussenvoegsel':...   
5857  [{'voornaam': 'Jan', 'tussenvoegsel': None, 'a...   
5875  [{'voornaam': 'Willem', 'tussenvoegsel': None,...   
5964  [{'voornaam': 'Pieter', 'tussenvoegsel': 'van'...   
6001  [{'voornaam': 'Willem', 'tussenvoegsel': 'de',...   
6044  [{'voornaam': 'Jacob', 'tussenvoegsel': None, ...   
6072  [{'voornaam': 'Michiel', 'tussenvoegsel': None...   
6106  [{'voornaam': 'Barend', 'tussenvoegsel': None,...   
6190  [{'voornaam': 'Jurgen', 'tussenvoegsel': None,...   

                                                   urls  \
5668  ['KLAB06472000152.JPG', 'KLAB06472000153.JPG',...   
5687  ['KLAB06472000078.JPG', 'KLAB06472000079.JPG',...   
5694  ['KLAB06472000201.JPG', 'KLAB06472000202.JPG',...   
5846     ['KLAB06471000222.JPG', 'KLAB06471000223.JPG']   
5857  ['KLAB06471000251.JPG', 'KLAB06471000252.JPG',...   
5875  ['KLAB06470000093.JPG', 'KLAB06470000094.JPG',...   
5964  ['KLAB06468000354.JPG', 'KLAB06468000355.JPG',...   
6001     ['KLAB06468000334.JPG', 'KLAB06468000335.JPG']   
6044  ['KLAB06467000112.JPG', 'KLAB06467000113.JPG',...   
6072  ['KLAB06467000295.JPG', 'KLAB06467000296.JPG',...   
6106  ['KLAB06467000073.JPG', 'KLAB06467000074.JPG',...   
6190  ['KLAB06693000219.JPG', 'KLAB06693000220.JPG

In [42]:
subset_thom

uuid  rubriek      notaris  \
54065  bbe2058e-6c82-c545-3128-233d2212f629      358  JAN VERLEIJ   
54066  cb4df4b2-8b3c-3ec8-8987-54f6b7628d47      358  JAN VERLEIJ   
54067  cb4df4b2-8b3c-3ec8-8987-54f6b7628d47      358  JAN VERLEIJ   
54068  cb4df4b2-8b3c-3ec8-8987-54f6b7628d47      358  JAN VERLEIJ   
54069  d5545553-6949-b0fa-04a2-dc50e6b71cc6      358  JAN VERLEIJ   
...                                     ...      ...          ...   
54208  4d0a5599-78bd-7040-e053-b784100a6264      358  JAN VERLEIJ   
54209  4d0a5599-78e5-7040-e053-b784100a6264      358  JAN VERLEIJ   
54210  4d0a5599-78e5-7040-e053-b784100a6264      358  JAN VERLEIJ   
54211  4d0a5599-78e5-7040-e053-b784100a6264      358  JAN VERLEIJ   
54212  4d0a5599-7903-7040-e053-b784100a6264      358  JAN VERLEIJ   

       inventarisNr  akteNr           akteType    datering        taal  \
54065         11901   65742         Machtiging  1747-07-20  nederlands   
54066         11901   65764  Scheepsverklaring  1747-05-02  nederlands   
54067         11901   65764  Scheepsverklaring  1747-05-02  nederlands   
54068         11901   65764  Scheepsverklaring  1747-05-02  nederlands   
54069         11901   65787          Kwitantie  1747-05-04  nederlands   
...             ...     ...                ...         ...         ...   
54208         11903   64981            Consent  1748-03-20  nederlands   
54209         11903   64991  Scheepsverklaring  1748-03-26  nederlands   
54210         11903   64991  Scheepsverklaring  1748-03-26  nederlands   
54211         11903   64991  Scheepsverklaring  1748-03-26  nederlands   
54212         11903   64997         Machtiging  1748-03-29  nederlands   

                                            beschrijving  \
54065                                               None   
54066         \nschip de Spaense Visser; Engelse kaper\n   
54067         \nschip de Spaense Visser; Engelse kaper\n   
54068         \nschip de Spaense Visser; Engelse kaper\n   
54069  \nschip de Vrouw Regina; in dienst overleden; ...   
...                                                  ...   
54208                                               None   
54209                                               None   
54210                                               None   
54211                                               None   
54212                                               None   

                                                   namen  \
54065  [{'voornaam': 'Doedje', 'tussenvoegsel': None,...   
54066  [{'voornaam': 'Gerrit', 'tussenvoegsel': None,...   
54067  [{'voornaam': 'Gerrit', 'tussenvoegsel': None,...   
54068  [{'voornaam': 'Gerrit', 'tussenvoegsel': None,...   
54069  [{'voornaam': 'Jan', 'tussenvoegsel': 'van de'...   
...                                                  ...   
54208  [{'voornaam': 'Hendrik', 'tussenvoegsel': None...   
54209  [{'voornaam': 'Roeloff', 'tussenvoegsel': None...   
54210  [{'voornaam': 'Roeloff', 'tussenvoegsel': None...   
54211  [{'voornaam': 'Roeloff', 'tussenvoegsel': None...   
54212  [{'voornaam': 'Auke', 'tussenvoegsel': None, '...   

                                                    urls  \
54065  ['A27120000399.JPG', 'A27120000400.JPG', 'A271...   
54066  ['A27120000013.JPG', 'A27120000014.JPG', 'A271...   
54067  ['A27120000013.JPG', 'A27120000014.JPG', 'A271...   
54068  ['A27120000013.JPG', 'A27120000014.JPG', 'A271...   
54069  ['A27120000020.JPG', 'A27120000021.JPG', 'A271...   
...                                                  ...   
54208     ['KLAB05521000188.JPG', 'KLAB05521000189.JPG']   
54209     ['KLAB05521000216.JPG', 'KLAB05521000217.JPG']   
54210     ['KLAB05521000216.JPG', 'KLAB05521000217.JPG']   
54211     ['KLAB05521000216.JPG', 'KLAB05521000217.JPG']   
54212  ['KLAB05521000231.JPG', 'KLAB05521000232.JPG',...   

                                                    text                name  \
54065  No: 297 Procuratie Gepasseert den 20e: Julij 1...        Doede Tonkes  

In [21]:
subset_chiel[subset_chiel.vocop_match != 0].iloc[7].text

"No: 333 Procuratie gecasseerd den 7e: Sept: 1348EOp heden den 7: Aepril des Jaars 1755 Compareerden Voor mij Jan Ver Notaris publiek bij den Ed: hole van hol Geadmitteert binnen Amsterdam Resider DE: Jans27e: Sipt 748 gCompareerden d' E. Jan Christoffel Helleff van Coppenhagen in 't Jaer 1746 voor bootsmans maat met het Schip Spaander wont voor de kamer Zeeland na Oostindien uijtgevaren en nu onlangs met 't Selve Schip in Gr: als onderstuurman voor de kamer Amsterdam gerepatrieert en behoorlijk affgedankt 5: de 1e: 0e: n & present binne dese Stadd De wel Verklaarde bij desen te Constiuueeren & magtieg te maken d' E. Dirk van den Ende wonende alhier omme in den naam & Van Weegens de Comparant te eijsschen Vorderen & ontfangen Van d' Ed: Heere Bewind hebbenen van de oostind: Compagnie ter kamere Amsterdam Sijn Comparants kist twee flessen kelders en twee potten alle gemerkt L: C en nog J: C6. mitsgaders nog om te verzoeken & ontfangen Sijn Comparants affreekening wegens voornoemde diens

In [16]:
subset_chiel[subset_chiel.vocop_match != 0]

uuid  rubriek      notaris  \
1399  4d0a5598-ea7b-7040-e053-b784100a6264      358  JAN VERLEIJ   
1415  4d0a5599-7775-7040-e053-b784100a6264      358  JAN VERLEIJ   
1416  4d0a5599-7775-7040-e053-b784100a6264      358  JAN VERLEIJ   
1463  3cb4f4c9-3056-3b97-c26f-7d010de31eae      358  JAN VERLEIJ   
1728  574e9081-263d-c1a6-db55-c1cb55e9a434      358  JAN VERLEIJ   
1729  574e9081-263d-c1a6-db55-c1cb55e9a434      358  JAN VERLEIJ   
1822  b011937e-a96a-8ce6-5856-37538debf093      358  JAN VERLEIJ   
1823  6d3a1c09-5c6f-ce55-77ae-d618e1c9d0f7      358  JAN VERLEIJ   
1825  6d3a1c09-5c6f-ce55-77ae-d618e1c9d0f7      358  JAN VERLEIJ   
1826  6d3a1c09-5c6f-ce55-77ae-d618e1c9d0f7      358  JAN VERLEIJ   

      inventarisNr  akteNr    akteType    datering        taal  \
1399         11903   59238   Testament  1748-01-06  nederlands   
1415         11903   64925  Machtiging  1748-01-15  nederlands   
1416         11903   64925  Machtiging  1748-01-15  nederlands   
1463         11903   75050  Attestatie  1748-02-10  nederlands   
1728         11905   13130  Machtiging  1748-09-19  nederlands   
1729         11905   13130  Machtiging  1748-09-19  nederlands   
1822         11905   18536  Machtiging  1748-11-29  nederlands   
1823         11905   18615  Machtiging  1748-09-07  nederlands   
1825         11905   18615  Machtiging  1748-09-07  nederlands   
1826         11905   18615  Machtiging  1748-09-07  nederlands   

                                           beschrijving  \
1399                                               None   
1415                                               None   
1416                                               None   
1463           \nVerklaren familierelaties te kennen.\n   
1728  \nschip de Eendragt, VOC kamer Amsterdam; schi...   
1729  \nschip de Eendragt, VOC kamer Amsterdam; schi...   
1822  \nVoor Kamer Amsterdam met schip Haerlmen naar...   
1823                                               None   
1825                                               None   
1826                                               None   

                                                  namen  \
1399  [{'voornaam': 'Coenraed Hendrik', 'tussenvoegs...   
1415  [{'voornaam': 'Claas', 'tussenvoegsel': None, ...   
1416  [{'voornaam': 'Claas', 'tussenvoegsel': None, ...   
1463  [{'voornaam': 'Pieter Cornelis', 'tussenvoegse...   
1728  [{'voornaam': 'Jan', 'tussenvoegsel': None, 'a...   
1729  [{'voornaam': 'Jan', 'tussenvoegsel': None, 'a...   
1822  [{'voornaam': 'Catharina', 'tussenvoegsel': No...   
1823  [{'voornaam': 'Abraham', 'tussenvoegsel': 'van...   
1825  [{'voornaam': 'Abraham', 'tussenvoegsel': 'van...   
1826  [{'voornaam': 'Abraham', 'tussenvoegsel': 'van...   

                                                   urls  \
1399  ['KLAB05521000021.JPG', 'KLAB05521000022.JPG',...   
1415     ['KLAB05521000061.JPG', 'KLAB05521000062.JPG']   
1416     ['KLAB05521000061.JPG', 'KLAB05521000062.JPG']   
1463  ['KLAB05521000125.JPG', 'KLAB05521000126.JPG',...   
1728  ['KLAB05559000104.JPG', 'KLAB05559000105.JPG',...   
1729  ['KLAB05559000104.JPG', 'KLAB05559000105.JPG',...   
1822  ['KLAB05559000570.JPG', 'KLAB05559000571.JPG',...   
1823  ['KLAB05559000020.JPG', 'KLAB05559000021.JPG',...   
1825  ['KLAB05559000020.JPG', 'KLAB05559000021.JPG',...   
1826  ['KLAB05559000020.JPG', 'KLAB05559000021.JPG',...   

                                                   text  \
1399  No: 6 Testament gepasseert e den 6e: Januarij ...   
1415  L: -N: 18 Procuratie gepasseert e den 15e: Jan...   
1416  L: -N: 18 Procuratie gepasseert e den 15e: Jan...   
1463  No: 38 Verklaring gepasseert den 10e: februari...   
1728  No: 360 Crocaratie gecasseerd den 19e: Sept: 1...   
1729  No: 360 Crocaratie gecasseerd den 19e: Sept: 1...   
1822  No: 507 Procuratie gepuisseert den 29e: Novemb...   
1823  No: 333 Procuratie gecasseerd den 7e: Sept: 13...   
1825  No: 333 Procuratie gecasseerd den 7e: Sept: 13...   
1826  No: 333 Procurat

#### Correcties:
Chiel:
- Christiaan Andriesz 1416
- Jan de Bruijn 1463
- Abraham van de Heuvel (1823 super interessant twijfel geval)
- Pieter van Essen (1825 super interessant twijfel geval)

Barry:
- Dirk Huijer 6044, mogelijke match met Dirk Huijer 76412